lib/Minify/ScssCssSource.php

```diff
@@ -0,0 +1,175 @@
+<?php
+use Leafo\ScssPhp\Compiler;
+use Leafo\ScssPhp\Server;
+use Leafo\ScssPhp\Version;
+
+/**
+ * Class for using SCSS files
+ *
+ * @link https://github.com/leafo/scssphp/
+ */
+class Minify_ScssCssSource extends Minify_Source {
```

Comment:
I think they both should rather depend on new Base Source class or the support that these classes have added to Source class. i.e some kind of cleanup/refactoring (in separate PR)